### ISSUE TITLE
:arg1-body with #() macro

### ISSUE THREAD

**Issue by dehli** [kkinnear/zprint#103]
First off, thanks for this great library! I'm really liking it! I've noticed that there seems to be incorrect behavior when using the `#()` macro along with `:arg1-body`. In my configuration I have `async` set as `:arg1-body`, so I'm seeing the below behavior (replaced spaces with `.`).

```
(async done
..(let [my-var ;; truncated
```

However, when I use `#()` I get the following behavior.

```
#(async done
..(let [my-var ;; truncated
```

I would expect there to be 3 spaces. Is this intended behavior? 

I'm using the latest native mac image (zprintm-0.4.15).

Thanks again!

**Comment by kkinnear** [comment]
You are absolutely correct, there should be three spaces there.  The problem is (now that you point it out), quite obvious in the code, and I will fix it.  I don't know how quickly I will be able to fix this, as I'm in the middle of an ever-expanding implementation of a `:respect-nl?`.  However, I'm thinking that I may need to cut a bug-fix release before I finish this feature, since it is taking a long time.  I don't have any workaround for this problem other than a fix. 

Thanks for noticing this.  I'm glad zprint is working out for you -- it is always good to hear that!

**Comment by dehli** [comment]
Sounds good! No rush on this. That `:respect-nl?` feature sounds neat though!

**Comment by kkinnear** [comment]
Fixed in version 0.4.16.
